resources/assets/coffee/react/beatmapset-page/scoreboard-tab.coffee

```diff
@@ -15,17 +15,14 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
 ###
-{span} = React.DOM
+{div} = React.DOM
 
-class BeatmapsetPage.ScoreboardTab extends React.Component
-  _scoreboardSwitch: =>
-    $.publish 'beatmapset:scoreboard:set', scoreboard: @props.scoreboard
+BeatmapsetPage.ScoreboardTab = (props) ->
+  className = 'beatmapset-scoreboard__tab page-tabs__tab'
```

Comment:
i think that less somehow can't extend from the nested selectors (like `&__tab` in this case)